.github/workflows/ci-dgraph-test-coverage.yml

```diff
@@ -0,0 +1,54 @@
+name: ci-dgraph-test-coverage
+on: [push, pull_request]
```

Comment:
I fixed this last evening, we are doing double triggers. 

you may want to change it to something like this https://github.com/dgraph-io/dgraph/pull/8241

Reply:
Got it, I've changed it in my latest commit. Thanks @skrdgraph

Reply:
resolving this conversation